### PR TITLE
Add missing confs

### PR DESCRIPTION
**Pull request by RubenRBS**
https://github.com/conan-io/conan/pull/14508
https://github.com/conan-io/conan/pull/14466
Among others

### CODE REVIEW

**Review by memsharded — APPROVED** [review]

**Review by memsharded — APPROVED** [review]

File: reference/commands/config.rst [hunk 1]
```diff
@@ -271,7 +275,7 @@ Displays all the Conan built-in configurations. There are 2 groups:
     tools.system.package_manager:mode: Mode for package_manager tools: 'check', 'report', 'report-installed' or 'install'
     tools.system.package_manager:sudo: Use 'sudo' when invoking the package manager tools in Linux (False by default)
     tools.system.package_manager:sudo_askpass: Use the '-A' argument if using sudo in Linux to invoke the system package manager (False by default)
-    tools.system.package_manager:tool: Default package manager tool: 'apk', 'apt-get', 'yum', 'dnf', 'brew', 'pacman', 'choco', 'zypper', 'pkg' or 'pkgutil'
+    tools.system.package_manager:tool: Default package manager tool: 'apt-get', 'yum', 'dnf', 'brew', 'pacman', 'choco', 'zypper', 'pkg' or 'pkgutil'
```
Comment (czoido) [inline reference/commands/config.rst:278]:
```suggestion
tools.system.package_manager:tool: Default package manager tool: 'apk', 'apt-get', 'yum', 'dnf', 'brew', 'pacman', 'choco', 'zypper', 'pkg' or 'pkgutil'
```

Comment (czoido) [inline reference/commands/config.rst:278]:
```suggestion
    tools.system.package_manager:tool: Default package manager tool: 'apk', 'apt-get', 'yum', 'dnf', 'brew', 'pacman', 'choco', 'zypper', 'pkg' or 'pkgutil'
```

File: reference/config_files/global_conf.rst [hunk 2]
```diff
@@ -118,7 +122,7 @@ To list all the possible configurations available, run :command:`conan config li
     tools.system.package_manager:mode: Mode for package_manager tools: 'check', 'report', 'report-installed' or 'install'
     tools.system.package_manager:sudo: Use 'sudo' when invoking the package manager tools in Linux (False by default)
     tools.system.package_manager:sudo_askpass: Use the '-A' argument if using sudo in Linux to invoke the system package manager (False by default)
-    tools.system.package_manager:tool: Default package manager tool: 'apk', 'apt-get', 'yum', 'dnf', 'brew', 'pacman', 'choco', 'zypper', 'pkg' or 'pkgutil'
+    tools.system.package_manager:tool: Default package manager tool: 'apt-get', 'yum', 'dnf', 'brew', 'pacman', 'choco', 'zypper', 'pkg' or 'pkgutil'
```
Comment (czoido) [inline reference/config_files/global_conf.rst:125]:
```suggestion
    tools.system.package_manager:tool: Default package manager tool: 'apk', 'apt-get', 'yum', 'dnf', 'brew', 'pacman', 'choco', 'zypper', 'pkg' or 'pkgutil'
```

File: reference/commands/config.rst (hunk 1 above)
Comment (czoido) [reply reference/commands/config.rst:278]: This reverts the latest changes in release/2.0 branch

Comment (czoido) [reply reference/commands/config.rst:278]: Looks like it was never updated to the list in release/2.0 and was manually added to the docs, will make a PR to fix it in Conan client.

**Review by czoido — APPROVED** [review]

**Pull request merged**